apps/src/code-studio/pd/workshop_dashboard/workshop.jsx

```diff
@@ -9,37 +9,15 @@ import _ from 'lodash';
 import PropTypes from 'prop-types';
```

Comment:
GitHub wants to hide this diff because it's too large, but please take a look - simplifying this component is why I think this change is worthwhile.